README.md

```diff
@@ -1,15 +1,15 @@
-# SpaCE
-SpaCE, the Spatial Confounding Environment, loads benchmark datasets for causal inference methods tackling spatial confounding.
+# spacebench
+spacebench, the Spatial Confounding Environment Benchmark, loads benchmark datasets for causal inference methods tackling spatial confounding.
```

Comment:
would keep original

Reply:
So Michelle, you are saying to do it like [`scikit-learn`](https://github.com/scikit-learn/scikit-learn)
where the repo and project are `scikit-learn` but the Python package is `sklearn`.